dynatrace/src/test/java/com/dynatrace/opentelemetry/metric/SerializerTest.java

```diff
@@ -0,0 +1,643 @@
+                    1619687659000000000L,
+                    Labels.empty(),
+                    Double.NEGATIVE_INFINITY));
+          }
+        };
+    DoubleSumData doubleSumData =
+        DoubleSumData.create(true, AggregationTemporality.CUMULATIVE, doublePointDataCollection);
+    MetricData metricData =
+        MetricData.createDoubleSum(
+            Resource.getDefault(),
+            InstrumentationLibraryInfo.empty(),
+            "doubleSumData",
+            "",
+            "",
+            doubleSumData);
+
+    List<String> doubleSumLines = serializer.createDoubleSumLines(metricData, false);
+    assertThat(doubleSumLines).hasSize(1);
+    assertThat(doubleSumLines.get(0)).isEqualTo("doubleSumData count,123.456");
+  }
+
+  @Test
+  void createDoubleSumLinesDelta() {
+    Collection<DoublePointData> doublePointDataCollection =
+        new ArrayList<DoublePointData>() {
+          {
+            add(
+                DoublePointData.create(
+                    1619687639000000000L, 1619687659000000000L, Labels.empty(), 123.456d));
+            add(
+                DoublePointData.create(
+                    1619687639000000000L, 1619687659000000000L, Labels.empty(), 321.456d));
+          }
+        };
+    DoubleSumData doubleSumData =
+        DoubleSumData.create(true, AggregationTemporality.DELTA, doublePointDataCollection);
+    MetricData metricData =
+        MetricData.createDoubleSum(
+            Resource.getDefault(),
+            InstrumentationLibraryInfo.empty(),
+            "doubleSumData",
+            "",
+            "",
+            doubleSumData);
+
+    List<String> doubleSumLines = serializer.createDoubleSumLines(metricData, true);
+    assertThat(doubleSumLines).hasSize(2);
+    assertThat(doubleSumLines.get(0)).isEqualTo("doubleSumData count,delta=123.456 1619687659000");
+    assertThat(doubleSumLines.get(1)).isEqualTo("doubleSumData count,delta=321.456 1619687659000");
+  }
+
+  @Test
+  void createDoubleSummaryLines() {
+    Collection<DoubleSummaryPointData> doubleSummaryPointDataCollection =
+        new ArrayList<DoubleSummaryPointData>() {
+          {
+            add(
+                DoubleSummaryPointData.create(
+                    1619687639000000000L,
+                    1619687659000000000L,
+                    Labels.empty(),
+                    7,
+                    700.70d,
+                    Arrays.asList(
+                        ValueAtPercentile.create(0.0, 0.1),
+                        ValueAtPercentile.create(100.0, 100.1))));
+            add(
+                DoubleSummaryPointData.create(
+                    1619687639000000000L,
+                    1619687659000000000L,
+                    Labels.empty(),
+                    3,
+                    660.66d,
+                    Arrays.asList(
+                        ValueAtPercentile.create(0.0, 0.22),
+                        ValueAtPercentile.create(100.0, 123.45))));
```

Comment:
Doesn't affect the unit test here but some of these don't add up and would thus be rejected by the server so they're probably not the best example.